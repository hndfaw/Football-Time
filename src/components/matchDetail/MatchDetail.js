import React from 'react';
import './matchDetail.css';


function MatchDetail({ homeTeamLogo, awayTeamLogo, homeTeamName, goalsHomeTeam, goalsAwayTeam, event_date, awayTeamName, status}) {

  return (
    <section className="match-detail-container">
      <div className="both-teams-container">
        <div className="single-team-container">
            <img src={homeTeamLogo} alt="home team logo" />
            <h4 className="detail-team-name">{homeTeamName}</h4>
        </div>
          <h4 className="detail-result"><span>{goalsHomeTeam}</span> - <span>{goalsAwayTeam}</span></h4>
        <div className="single-team-container">
            <img src={awayTeamLogo} alt="away team logo" />
            <h4 className="detail-team-name">{awayTeamName}</h4>
        </div>
      </div>
        <h5 className="detail-other-info"><span>{status}</span> | <span>{event_date}</span></h5>

    </section>
    
  )
}


export default  MatchDetail
