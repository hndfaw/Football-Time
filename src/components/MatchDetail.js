import React from 'react';
import './matchDetail.css'

function MatchDetail({fixture_id}) {
  console.log(fixture_id)
  return (
    <div>
      <h1 className="text">{fixture_id}</h1>
    </div>
  )
}


export default  MatchDetail
