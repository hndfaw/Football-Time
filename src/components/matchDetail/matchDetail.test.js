import React from 'react';
import { shallow } from 'enzyme';
import { MatchDetail } from './MatchDetail';



describe('Home', () => {
  
    it('should match the match snapshot', () => {
      let wrapper = shallow(<MatchDetail  />)
      expect(wrapper).toMatchSnapshot()
    })
})