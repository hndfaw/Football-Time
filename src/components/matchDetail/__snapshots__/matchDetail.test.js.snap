// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match the match snapshot 1`] = `
<section
  className="match-detail-container"
>
  <div
    className="both-teams-container"
  >
    <div
      className="single-team-container"
    >
      <img
        alt="home team logo"
      />
      <h4
        className="detail-team-name"
      />
    </div>
    <h4
      className="detail-result"
    >
      <span />
       - 
      <span />
    </h4>
    <div
      className="single-team-container"
    >
      <img
        alt="away team logo"
      />
      <h4
        className="detail-team-name"
      />
    </div>
  </div>
  <h5
    className="detail-other-info"
  >
    <span />
     | 
    <span />
  </h5>
</section>
`;
